client/components/file-picker/docs/example.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+
+export default class FilePickers extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.onSingle = this.onSingle.bind( this );
+		this.onMulti = this.onMulti.bind( this );
+	}
+
+	onSingle( files ) {
+		alert( 'Selected file: ' + JSON.stringify( files[0].name ) );
+	}
+
+	onMulti( files ) {
+		alert( 'Selected files:\n' + [].slice.call( files ).map( ( file ) => {
+			return '  ' + JSON.stringify( file.name );
+		} ).join( '\n' ) );
+	}
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/file-pickers">File Picker</a>
+				</h2>
+				{ this.renderPickers() }
+			</div>
+		);
+	}
+
+	renderPickers() {
+		return (
+			<Card>
+
+				<h4>Select a single file:</h4>
```

Comment:
Hmmm, I was under the impression that [docs don't get translated](https://github.com/Automattic/wp-calypso/pull/1468#issuecomment-164364801). Is this just for good practice?